dsl/src/main/java/tech/pegasys/peeps/node/model/GenesisAddress.java

```diff
@@ -63,8 +63,7 @@ public boolean equals(Object obj) {
     if (getClass() != obj.getClass()) return false;
     GenesisAddress other = (GenesisAddress) obj;
     if (hex == null) {
-      if (other.hex != null) return false;
-    } else if (!hex.equals(other.hex)) return false;
-    return true;
+      return other.hex == null;
+    } else return hex.equals(other.hex);
```

Comment:
nit: `else {
      return ... ;
      }`
use brackets for else condition.

Reply:
done. reverted that change. didn't mean to mess with the equals